runtime/compiler/codegen/J9RecognizedMethodsEnum.hpp

```diff
@@ -426,6 +426,8 @@
 
    jdk_internal_misc_Unsafe_copyMemory0,
 
+   jdk_internal_loader_NativeLibraries_load,
```

Comment:
This isn't ifdeffed, but everything else seems to be.

Reply:
Yeah, somehow `#if JAVA_SPEC_VERSION >= 15` doesn't work in this file (haven't figured out yet)
On the other hand, I guess it won't hurt much to add an extra enum entry for pre-JDK15 levels.

Reply:
It's a matter of whether the data has to match the enum order. If so, it all needs to be ifdeffed, or none of it. @andrewcraik can best answer this.

Reply:
It could be as simple as a failure to `#include "j9cfg.h"`.

Reply:
the enum entries are not conditional - you will add the recognized method entry for all versions of the JDK - it will only match something on a newer version of the JVM. The JIT tries to avoid version ifdefs if at all possible.

Reply:
> It could be as simple as a failure to #include "j9cfg.h".

No, `J9RecognizedMethodsEnum.hpp` doesn't take `j9cfg.h` well.
```
build/macosx-x86_64-server-release/vm/compiler/../include/j9cfg.h:32:1: error: expected identifier
extern "C" {
```
The `#include` in question already presents in `j9method.cpp` though it still can't find the enum entry.
```
compiler/env/j9method.cpp:3389:14: error: no member named
      'jdk_internal_loader_NativeLibraries_load' in namespace 'TR'
```

Anyway, there is no need to add `ifdefs` here as per https://github.com/eclipse/openj9/pull/11160#discussion_r522313553.